diff --git a/docs/source/automod.py b/docs/source/automod.py
index 0fb4839..e1e66bb 100644
--- a/docs/source/automod.py
+++ b/docs/source/automod.py
@@ -50,7 +50,7 @@ class_exprs = { "ObjectID": "h5py.h5.ObjectID",
 
 class_exprs = dict( 
     (re.compile(class_base % x.replace(" ",r"\s"), re.VERBOSE), y) \
-    for x, y in class_exprs.iteritems() )
+    for x, y in class_exprs.items() )
 
 def replace_class(istr):
 
@@ -58,7 +58,7 @@ def replace_class(istr):
         pre, name, post = match.group('pre', 'name', 'post')
         return '%s:class:`%s <%s>`%s' % (pre, name, target, post)
 
-    for expr, target in class_exprs.iteritems():
+    for expr, target in class_exprs.items():
         rpl2 = partial(rpl, target)
         istr = safe_replace(istr, expr, rpl2)
 
